abecto/vocabulary/README.md

```diff
@@ -0,0 +1,28 @@
+# /abecto/vocabulary
+
+This W3ID provides a persistent URI namespace for the [ABECTO Vocabulary (AV)](https://github.com/fusion-jena/abecto-vocabulary).
+
+## Uses
+
+Provides access to the HTML and RDF representations of the latest and specific version of the vocabulary.
+
+## Example URLs
+
+* Latest version, whole vocabulary: http://w3id.org/abecto/vocabulary
+* Latest version, specific resource: http://w3id.org/abecto/vocabulary#Plan
+* Specific version, whole vocabulary:http://w3id.org/abecto/vocabulary/0.1
+* Specific version, specific resource: http://w3id.org/abecto/vocabulary/0.1#Plan
```

Comment:
```suggestion
* Latest version, whole vocabulary: <http://w3id.org/abecto/vocabulary>
* Latest version, specific resource: <http://w3id.org/abecto/vocabulary#Plan>
* Specific version, whole vocabulary: <http://w3id.org/abecto/vocabulary/0.1>
* Specific version, specific resource: <http://w3id.org/abecto/vocabulary/0.1#Plan>
```
Adds omitted space; wraps URLs in angle-brackets which will not be visible when markdown is processed into HTML for rendering & display by browsers, etc.